### PR TITLE
Added test LG7

### PR DESCRIPTION
**Pull request by RoyVanVu**
# Description

Creating test LG7 from the design document in collaboration with https://github.com/VivianBelle17 and https://github.com/Vizsla1
Partly Addresses: #962

## Type of change

- [ ] Note merging this changes the database configuration.
- [ ] This change requires a documentation update

## Checklist

- [x] I have followed the [OED pull request](https://openenergydashboard.github.io/developer/pr.html) ideas
- [x] I have removed text in ( ) from the issue request
- [x] You acknowledge that every person contributing to this work has signed the [OED Contributing License Agreement](https://openenergydashboard.github.io/developer/cla.html) and each author is listed in the Description section.

## Limitations

First time use docker and find a way to go through all of the files

### CODE REVIEW

**Comment by huss** [comment]
Thanks to @RoyVanVu  for this contribution. I don't see a record that you signed the CLA. Assuming this is correct, please use the link in the description/Checklist to do this. If you think my records are wrong then please let me know. Also, there are often teams working on the tests so if others contributed to this work then please let me know their GitHub IDs (often put in the pull request description) and get them to do the CLA. If not, just let me know you worked on this alone. Please let me know if I can help.

**Comment by RoyVanVu** [comment]
Hi @huss! I'm sorry for the issue. My team member has completed the CLA agreement, and her GitHub name is VivianBelle17 (https://github.com/VivianBelle17). I hope this will help; please don't hesitate to let me know if there are any other issues.

**Comment by huss** [comment]
@RoyVanVu I now see a CLA from everyone but you. If that is correct, could you please fill it out. Let me know if you have any thoughts/questions.

**Comment by RoyVanVu** [comment]
Hi @huss! I finish my CLA agreement

File: src/server/test/web/readingsLineGroupQuantity.js [hunk 1]
```diff
@@ -103,6 +103,20 @@ mocha.describe('readings API', () => {
                 });
 
                 // Add LG7 here
```
Comment (huss) [inline src/server/test/web/readingsLineGroupQuantity.js]: Could you please remove this line with a pladeholder comment since you have added that test.

**Review by huss — CHANGES_REQUESTED** [review]
Thanks to @RoyVanVu, @VivianBelle17 & @Vizsla1 for their first contribution to OED. There is one small comment to address. Also, the needed expected file (src/server/test/web/readingsData/expected_line_group_ri_15-20_mu_kWh_gu_kWh_st_2022-08-20%07#25#35_et_2022-10-28%13#18#28.csv) is not in the files changed. It needs to be included so it is part of the GitHub repository. Please let me know if you have questions/thoughts.

**Comment by RoyVanVu** [comment]
Hi @huss! I just uploaded the expected_line_group_ri_15-20_mu_kWh_gu_kWh_st_2022-08-20%07#25#35_et_2022-10-28%13#18#28.csv and deleted the line of the comment above LG7 as you said

**Comment by huss** [comment]
The changes look good. Congratulations to  @RoyVanVu, @VivianBelle17 & @Vizsla1 on their first merged contribution.

**Review by huss — APPROVED** [review]

**Pull request merged**